Internship2022TeamPatri/Internship2022TeamPatri/Views/Auth/Login/LogIn.storyboard

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="19455" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="3t1-SN-qvL">
+    <device id="retina6_7" orientation="portrait" appearance="light"/>
+    <dependencies>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="19454"/>
```

Comment:
Do we really need to use a storyboard for Login?

Reply:
I completely agree with Camelia, storyboards should be used for more complex flows. Instead, you could try xib / programmatic code